students/mihwa-yang/users/views.py

```diff
@@ -37,4 +37,21 @@ def post(self, request):
             nickname = data.get('nickname'),
             phone    = data.get('phone')
         )
-        return JsonResponse({'MESSAGE':'SIGNUP SUCCESS'}, status=201)
\ No newline at end of file
+        return JsonResponse({'MESSAGE':'SIGNUP SUCCESS'}, status=201)
+
+
+class SignInView(View): 
+    def post(self, request):
+        data  = json.loads(request.body)
+
+        if 'email' not in data.keys() or 'password' not in data.keys():
+            return JsonResponse({"MESSAGE": "KEY ERROR"}, status=400)
+
+        if User.objects.filter(email=data['email']).exists() == False:
```

Comment:
```suggestion
        if not User.objects.filter(email=data['email']).exists():
```
.exists()의 결과 값이 boolean이기 때문에 위와 같이 사용하는 것이 좋습니다.

Reply:
넵 수정했습니다 👍